mof_array/pmf/process_mass_data.py

```diff
@@ -1,31 +1,44 @@
-"""This code imports mass adsorption data and "experimental" adsorption data
-from simulated output, for multiple MOFs and gas mixtures, and calculates the
-probability that specific gases are present in specific mole fractions
-(in each experimental case) based on the total mass adsorbed for each MOF.
-Additionally, the best MOF arrays for detecting each gas are reported,
-according to the highest information gain.
-"""
```

Comment:
keep the herdoc format so the help strings work. ok to do by adding one additional commit